map/query-features-with-arcade-expression/src/main/java/com/esri/samples/query_features_with_arcade_expression/QueryFeaturesWithArcadeExpressionSample.java

```diff
@@ -0,0 +1,197 @@
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.query_features_with_arcade_expression;
+
+import java.util.HashMap;
+
+import javafx.application.Application;
+import javafx.geometry.Point2D;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ProgressIndicator;
+import javafx.scene.input.MouseButton;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment;
+import com.esri.arcgisruntime.arcade.ArcadeEvaluationResult;
+import com.esri.arcgisruntime.arcade.ArcadeEvaluator;
+import com.esri.arcgisruntime.arcade.ArcadeExpression;
+import com.esri.arcgisruntime.arcade.ArcadeProfile;
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.data.ArcGISFeature;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.layers.Layer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.view.Callout;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.portal.PortalItem;
+
+public class QueryFeaturesWithArcadeExpressionSample extends Application {
+
+  private MapView mapView;
+  private ArcGISMap map;
+  private Layer policeBeatsLayer;
+  private final String rpdBeatsLayerName = "RPD Beats  - City_Beats_Border_1128-4500";
+  private final String calloutText = "Crimes in the last 60 days: ";
+
+  @Override
+  public void start(Stage stage) {
+    try {
+      // create stack pane and application scene
+      var stackPane = new StackPane();
+      var scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Query Features With Arcade Expression Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a progress indicator
+      var progressIndicator =  new ProgressIndicator();
+
+      // create a portal item and use it to create a map
+      var portal = new Portal("https://www.arcgis.com/");
+      var portalItem = new PortalItem(portal, "539d93de54c7422f88f69bfac2aebf7d");
+      map = new ArcGISMap(portalItem);
+
+      // create a map view and set the map to it
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // set callout leader to appear at the bottom of the callout
+      mapView.getCallout().setLeaderPosition(Callout.LeaderPosition.BOTTOM);
```

Comment:
we can instantiate the `Callout` here to save repeat usages of `mapView.getCallout()`, and apply that to the code where the detail and title are being set

```suggestion
      // set callout leader to appear at the bottom of the callout
      Callout callout = mapView.getCallout();
      callout.setLeaderPosition(Callout.LeaderPosition.BOTTOM)
```